app/models/item.rb

```diff
@@ -51,4 +52,13 @@ def location_name
     "Unknown"
   end
 
+  def repair_condition
```

Comment:
I would like this to be an active verb, more descriptive. Imagine yourself seeing it in 6 months. What does it do ?
Perhaps `set_repair_condition` ?